File: python/robotmap.py
# Imports
import wpilib
import rev

# Define constants
brushless = rev.MotorType.kBrushless

class RobotMap():
    motors = {
        "leftFront": rev.CANSparkMax(1, brushless),
        "leftCenter": rev.CANSparkMax(2, brushless),
        "leftBack": rev.CANSparkMax(3, brushless),
        "rightFront": rev.CANSparkMax(4, brushless),
        "rightCenter": rev.CANSparkMax(5, brushless),
        "rightBack": rev.CANSparkMax(6, brushless)
    }

    leftDrive = wpilib.SpeedControllerGroup(motors["leftFront"], motors["leftCenter"], motors["leftBack"])
    rightDrive = wpilib.SpeedControllerGroup(motors["rightFront"], motors["rightCenter"], motors["rightBack"])

File: billyGoesToSpace/src/main/java/billy/robot/RobotMap.java
package billy.robot;

public class RobotMap {
  public enum Controllers {
    MAIN_XBOX(0), BUTTONS(1), XBOX_2(2);

    public int port;

    Controllers(int port) {
      this.port = port;
    }
  }
  public enum CANDevice {
    LEFT_BACK(1), LEFT_CENTER(2), LEFT_FRONT(3),
    RIGHT_BACK(4), RIGHT_CENTER(5), RIGHT_FRONT(6);

    public int id;

    CANDevice(int id) {
      this.id = id;
    }
  }

}
